### PR TITLE
non blocking database

### PR DESCRIPTION
**Pull request by Taza53**
Non-blocking new thread DB

### CODE REVIEW

File: webserver/src/macros_local.rs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#[macro_export]
+macro_rules! get_db_connection {
+    ($pool:expr) => {{
```
Comment (dIB59) [inline webserver/src/macros_local.rs:3]: Are macros necessary. Might make the code hard to read for newer people.

Comment (Taza53) [reply webserver/src/macros_local.rs:3]: Not necessary, just cleaner and shorter.

**Comment by Taza53** [comment]
@dIB59 needs review.

**Review by dIB59 — APPROVED** [review]

**Review by dIB59 — APPROVED** [review]
LGTM

**Pull request merged**